src/plugins/dashboard/common/embeddable/dashboard_container_persistable_state.ts

```diff
@@ -22,6 +22,7 @@ import { CONTROL_GROUP_TYPE } from '../../../controls/common';
 const getPanelStatePrefix = (state: DashboardPanelState) => `${state.explicitInput.id}:`;
 
 const controlGroupReferencePrefix = 'controlGroup_';
+const controlGroupId = 'dashboard_control_group';
```

Comment:
This is a funny change...

The `persistableStateService` requires an id in its input because it requires a full instance of the generic `embeddableInput`. The `embeddableId` in the generic `embeddableInput` is required in order to differentiate multiple instances of the same embeddable in a container. Because the Control Group on a dashboard is a singleton, the ID isn't used for anything, or saved into the dashboard saved object.

Before this PR, the ID was technically missing but since the type required an `id` key even though it was never used, it worked correctly. I switched out the types in this PR to make everything more consistent, and it required that I pass in an id, so I passed in a generic const.

Reply:
Ahhhh, okay okay. That makes a lot more sense. Thanks for the clarification 👍